sass/_clients.scss

```
@@ -1,77 +1,161 @@
-.featured-clients {
+.hero-header {
+    text-align: center;
+    margin-top: 90px;
+    margin-bottom: 81px;
+    color: #000000;
+    width: 100%;
+    max-width: 651px;
+
+
+    @media (max-width: 767px) {
+        max-width: 312px;
+        margin-top: 56px;
+        margin-bottom: 34px;
+    }
+
+    h1 {
+        width: 100%;
+        font-style: normal;
+        font-weight: 700;
+        font-size: 40px;
+        line-height: 120%;
+    }
+
+    p {
+        width: 100%;
+        font-style: normal;
+        font-weight: 400;
+        font-size: 18px;
```

I would use `rem` here as well

```suggestion
        font-size: 1.125rem;
```